lighthouse-core/report/html/renderer/dom.js

```diff
@@ -18,6 +18,8 @@
 
 /* globals URL self */
 
+/** @typedef {HTMLElementTagNameMap & {[id: string]: HTMLElement}} HTMLElmentByTagName */
```

Comment:
`HTMLElementTagNameMap` is the built-in you're talking about?

Reply:
> `HTMLElementTagNameMap` is the built-in you're talking about?

yes